armi/utils/tests/test_gridGui.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google, LLC
+# Copyright 2021 TerraPower, LLC
```

Comment:
not sure if we can actually do this. original test was from @scottyak at Google. I think if you've added something you can put TerraPower in there, too, but dont think we can remove Google. Might do some research to be sure.

Reply:
OH, is that what happened? 

I just assumed this was a typo.

Google has partially ownership of the copyright of ARMI?  That doesn't seem right.

So, any developer on ARMI can copyright a portion of the codebase for themselves?  That's problematic.  What do we do if a developer (or in this case Google) decides to flex their copyright?

This is honestly much stranger than what I originally assumed was just a typo. huh.

Reply:
As I said, I thought this was just a typo. I can change it back.